templates/course/lesson-preview.php

```diff
@@ -46,6 +47,9 @@
 				<h6 class="llms-pre-text"><?php echo $pre_text; ?></h6>
 			<?php endif; ?>
 			<h5 class="llms-h5 llms-lesson-title"><?php echo get_the_title( $lesson->get( 'id' ) ); ?></h5>
+			<?php if ( 'course' === get_post_type( get_the_ID() ) && apply_filters( 'lifterlms_single_course_syllabus_lesson_favorite', true ) ) : ?>
```

Comment:
1) missing documentation for the filter
2) the filter prefix should be `llms_` not `lifterlms_`
3) Also about the naming `lifterlms_single_course_syllabus_lesson_favorite` ... Can you make it more descriptive of what it's going to do? It seems that filters the lesson's favorite, but it filters whether or not we're gonna show the favorite...
It's a bit different.

Reply:
fixed.

I've renamed the hook to `llms_course_syllabus_lesson_favorite_visibility`.